packages/engine/Source/Scene/Scene.js

```diff
@@ -1880,9 +1898,16 @@ Scene.prototype.updateFrameState = function () {
   frameState.cameraUnderground = this._cameraUnderground;
   frameState.globeTranslucencyState = this._globeTranslucencyState;
 
-  if (defined(this.globe)) {
-    frameState.terrainExaggeration = this.globe.terrainExaggeration;
-    frameState.terrainExaggerationRelativeHeight = this.globe.terrainExaggerationRelativeHeight;
+  frameState.verticalExaggeration = this.verticalExaggeration;
+  frameState.verticalExaggerationRelativeHeight = this.verticalExaggerationRelativeHeight;
+  const { globe } = this;
+  if (defined(globe) && globe._terrainExaggerationChanged) {
```

Comment:
Question: if old code uses `globe.terrainExaggeration` but not the new `scene.terrainExaggeration`, wouldn't this only take effect for the first frame that the variable was set, then the next frame it would be clobbered by the lines above this?

I'm asking because for #4196 I might need to do something similar for deprecating some of the atmosphere properties.

Reply:
Good catch, thanks!